controllers/binding/snapshotenvironmentbinding_adapter.go

```diff
@@ -15,15 +15,16 @@ package binding
 
 import (
 	"context"
+
 	"github.com/redhat-appstudio/operator-goodies/reconciler"
 	ctrl "sigs.k8s.io/controller-runtime"
 
-	"github.com/go-logr/logr"
 	applicationapiv1alpha1 "github.com/redhat-appstudio/application-api/api/v1alpha1"
 	"github.com/redhat-appstudio/integration-service/api/v1alpha1"
 	"github.com/redhat-appstudio/integration-service/gitops"
-	"github.com/redhat-appstudio/integration-service/helpers"
+	h "github.com/redhat-appstudio/integration-service/helpers"
```

Comment:
I found in some files the import is change `h "github.com/redhat-appstudio/integration-service/helpers"`,  is there any special reason? Thanks.

Reply:
just to make it shorter in the code

Reply:
I guess it is the reason. Just notice in some file it still use the old style, such as https://github.com/redhat-appstudio/integration-service/pull/160/files#diff-8147062616b315a0724a1e97be5d43ac179afd264b847c27fb31d21842f308f3R34, so I wonder. But it doesn't matter. Thanks for the explanation.